src/Http/Headers/src/PublicAPI.Unshipped.txt

```diff
@@ -1 +1,2 @@
 #nullable enable
+static Microsoft.Net.Http.Headers.HeaderUtilities.IsQuoted(Microsoft.Extensions.Primitives.StringSegment input) -> bool
```

Comment:
Revert

Reply:
I'm now adding a new FormatInt64 method, I reverted back this line and added a new one for the new method. Is that ok?